infra/arm-templates/storage.template.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Should we update the schemas for the rest of ARM templates also or are those fine?

Reply:
sounds good then, thanks for working on this!